README.md

```diff
@@ -98,12 +98,16 @@ The following packages are optional:
 
  More font support in the reports
 
-* **gir1.2-geocodeglib-1.0**
+* **geocodeglib**
 
  A library use to associate a geographical position (latitude, longitude)
  to a place name. This is used if you already have osmgpsmap installed.
  If installed, when you add or link a place from the map, you have a red line
  at the end of the table for selection.
+ Debian, Ubuntu, ... : gir1.2-geocodeglib-1.0
+ Fedora, Redhat, ... : ?
```

Comment:
Probably [geocode-glib](https://apps.fedoraproject.org/packages/geocode-glib) as well.

Reply:
I look at in this rpm, but the gir file is missing. So we need another rpm which provides ...

Ah yes, new versions doesn't need the gir file. only the typelib is necessary.
This rpm provides : 
/usr/lib/girepository-1.0/GeocodeGlib-1.0.typelib
So it's good. I modify the readme file.